aggsender/db/aggsender_db_storage.go

```diff
@@ -268,8 +275,34 @@ func (a *AggSenderSQLStorage) SaveOrUpdateCertificate(ctx context.Context, certi
 	return nil
 }
 
+// saveSignedCertificateToFile saves the signed certificate content to a file in the same directory as the database
+// and returns the file path
+func (a *AggSenderSQLStorage) saveSignedCertificateToFile(
+	certificateID common.Hash,
+	signedCertContent string) (string, error) {
+	// Get the directory where the database is stored
+	dbDir := filepath.Dir(a.cfg.DBPath)
+
+	// Create a filename using the certificate ID
+	fileName := fmt.Sprintf("signed_cert_%s.json", hex.EncodeToString(certificateID[:]))
```

Comment:
Nitpick: it should be enough to simplify it this way and let the `Stringer` implementation do it's stuff. WDYT?

```suggestion
	fileName := fmt.Sprintf("signed_cert_%s.json", certificateID)
```